interface/i18n/mist.nl.i18n.json

```diff
@@ -218,8 +252,12 @@
                 "you": "Je",
                 "etherbase": "Centrale account (Etherbase)",
                 "depositBitcoin": "Bitcoin Storten",
+                "viaShapeshift": "Direct omwisselen via <a href='https://shapeshift.io/' target='_blank'>Shapeshift</a>",
                 "learnIt": "Leer terwijl je wacht",
                 "downloadingBlocks": "Blokken downloaden",
+                "syncMessage": "Blok __displayBlock__ van __highestBlockString__",
+                "syncMessageWithStates": "Blok __displayBlock__ van __highestBlockString__ (Chain structuur __statesPercent__%)",
+                "startingSync": "Gereed maken voor sync..",
```

Comment:
Probably something like `Wachten op synchronisatie...` is better.
But it depends on the context.